frontend/packages/ux-editor/src/components/leftMenu/LayoutSetsContainer.test.tsx

```diff
@@ -18,26 +16,46 @@ describe('LayoutSetsContainer', () => {
     render();
 
     expect(
-      await screen.findByRole('button', { name: layoutSetsMock.sets[0].id })
+      await screen.findByRole('option', { name: layoutSetsMock.sets[0].id }),
     ).toBeInTheDocument();
     expect(
-      await screen.findByRole('button', { name: layoutSetsMock.sets[1].id })
+      await screen.findByRole('option', { name: layoutSetsMock.sets[1].id }),
     ).toBeInTheDocument();
   });
 
-  it('updates selected layout when clicking on layout button', async () => {
-    const mockDispatch = jest.fn();
-    (useDispatch as jest.Mock).mockReturnValue(mockDispatch);
+  it('NativeSelect should be rendered', async () => {
+    render();
+    expect(screen.getByRole('combobox')).toBeInTheDocument();
+  });
 
+  it('calls dispatch when selecting an option', async () => {
+    const dispatch = jest.fn();
+    (useDispatch as jest.Mock).mockReturnValue(dispatch);
     render();
 
-    const button = await screen.findByRole('button', { name: layoutSetsMock.sets[0].id });
-    await act(() => user.click(button));
+    await waitFor(async () => {
+      await userEvent.selectOptions(
+        screen.getByRole('combobox'),
+        screen.getByRole('option', { name: layoutSetsMock.sets[0].id }),
+      );
+    });
+
+    expect(dispatch).toHaveBeenCalledTimes(1);
```

Comment:
Det er veldig bra å verifisere at funksjonen bare blir kalt én gang, men du kan godt beholde testen som var her også, så vi fortsatt får verifisert at den blir kalt med riktig verdi.
```suggestion
    expect(dispatch).toHaveBeenCalledTimes(1);
    expect(dispatch).toHaveBeenCalledWith({
      payload: layoutSetsMock.sets[0].id,
      type: 'formDesigner/updateSelectedLayoutSet',
    });
```